<launch>
  <arg name="dataset_path"   default="/home/costar/Downloads/LoopComputeBenchmark" />
  <arg name="gt_odom_bag"    default="odometry.bag" />
  <arg name="lamp_bag"       default="base_input.bag" />
  <arg name="robot_name"     default="husky1" />
  <arg name="radius_tol"     default="0.3" />
  <arg name="gt_odom_topic"  default="/$(arg robot_name)/lo_frontend/odometry" />

  <!-- Base station LAMP -->
  <node pkg="loop_closure"
        name="gen_loop_computation_test"
        type="gen_loop_computation_test"
        output="screen"> 
    <param name="dataset_path"   value="$(arg dataset_path)" />
    <param name="gt_odom_bag"    value="$(arg gt_odom_bag)" />
    <param name="gt_odom_topic"  value="$(arg gt_odom_topic)" />
    <param name="lamp_bag"       value="$(arg lamp_bag)" />
    <param name="robot_name"     value="$(arg robot_name)" />
    <param name="radius_tol"     value="$(arg radius_tol)" />
  </node>

</launch>
